script/foreman-debug

```diff
@@ -287,7 +287,9 @@ add_cmd "gem list" "gem_list"
 add_cmd "scl enable $SCLNAME 'gem list'" "gem_list_scl"
 add_cmd "bundle --local --gemfile=/usr/share/foreman/Gemfile" "bundle_list"
 add_cmd "facter" "facts"
-add_files /etc/foreman/* /var/log/foreman/*.log*
+# exclude *key.pem files and encryption_key.rb
+add_files /etc/foreman/*.{yml,yaml,conf} /etc/foreman/*{cert,ca}.pem /etc/foreman/plugins/
```

Comment:
1. No need for *{cert,ca}.pem here I think, that depends on the user's configuration? If it's from Katello, please add it to their debug scripts.
1. /etc/foreman/plugins/ is missing a filename, so it's not picking up any files. *.yaml might be most precise (that's all that's loaded anyway).